app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { Checkbox, Button, Tooltip, Avatar, Alert } from "neetoui";
+
+import { DELETE_CONTACT_ALERT_MSG } from "./Constants";
+
+export default function ContactTable({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+}) {
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+
+  return (
+    <div className="w-full px-4">
+      <table className="nui-table nui-table--checkbox">
```

Comment:
Please add classnames `nui-table--hover nui-table--actions`. 
These classnames would add hover effects to the table row and IconButtons respectively.